added note about docker tag command

ealvesss opened the pull request:
Just a note in case the user is using your own image, they have to tag the image with de registry url.

azurecla commented:
Hi __@NingKuang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented on articles/virtual-machines-linux-mysql-use-opensuse.md:
```diff
@@ -14,7 +14,7 @@
 	ms.devlang="na"
 	ms.topic="article"
-	ms.date="12/12/2014"
+	ms.date="21/05/2015"
```
@NingKuang Could you please change this to MM/DD/YYYY format?

ealvesss committed:
added note about docker tag command

Just a note in case the user is using your own image, they have to tag the image with de registry url.

GitHubber17 committed:
Update tutorial-custom-docker-image.md
